src/main/java/com/github/vkorobkov/jfixturescmd/CmdArgs.java

```diff
@@ -0,0 +1,17 @@
+package com.github.vkorobkov.jfixturescmd;
+
+import com.beust.jcommander.Parameter;
+import lombok.Getter;
+
+@Getter
+final class CmdArgs {
```

Comment:
Can we have blank lines between the class fields? Just to improve readability a bit